substra/cli/printers.py

```diff
@@ -264,7 +264,7 @@ class BaseAlgoPrinter(AssetPrinter):
 class ComputePlanPrinter(AssetPrinter):
     asset_name = 'compute_plan'
 
-    key_field = Field('Compute plan ID', 'compute_plan_id')
+    key_field = Field('Compute Plan key', 'key')
```

Comment:
You could remove this line entirely, it would fallback onto the `key_field = Field('Key', 'key')` (similar to what is done for other fields)